Reports/report-docx-dailyIncidentReport.json

```diff
@@ -1,396 +1,821 @@
+            ]
+          },
+          "query": "",
+          "modified": "2018-01-25T10:00:19.85150374Z",
+          "name": "dailyIncidentReport Type",
+          "dateRange": {
+            "fromDate": "0001-01-01T00:00:00Z",
+            "toDate": "0001-01-01T00:00:00Z",
+            "period": {
+              "byTo": "",
+              "byFrom": "days",
+              "toValue": null,
+              "fromValue": null,
+              "field": ""
+            },
+            "fromDateLicense": "0001-01-01T00:00:00Z"
+          },
+          "isPredefined": false,
+          "version": 1,
+          "id": "111",
+          "widgetType": "pie"
+        }
+      },
+      {
+        "id": "665f6190-01b7-11e8-9918-2d303bb12520",
+        "forceRange": false,
+        "x": 0,
+        "y": 4,
+        "i": "665f6190-01b7-11e8-9918-2d303bb12520",
+        "w": 6,
+        "h": 2,
+        "widget": {
+          "size": 0,
+          "dataType": "incidents",
+          "params": {
+            "groupBy": [
+              "dbotDueDate"
+            ]
+          },
+          "query": "",
+          "modified": "2018-01-25T10:00:29.171831558Z",
+          "name": "dailyIncidentReport SLA",
+          "dateRange": {
+            "fromDate": "0001-01-01T00:00:00Z",
+            "toDate": "0001-01-01T00:00:00Z",
+            "period": {
+              "byTo": "",
+              "byFrom": "days",
+              "toValue": null,
+              "fromValue": null,
+              "field": ""
+            },
+            "fromDateLicense": "0001-01-01T00:00:00Z"
+          },
+          "isPredefined": false,
+          "version": 1,
+          "id": "112",
+          "widgetType": "pie"
+        }
+      },
+      {
+        "id": "66e840a0-01b7-11e8-9918-2d303bb12520",
+        "forceRange": false,
+        "x": 6,
+        "y": 4,
+        "i": "66e840a0-01b7-11e8-9918-2d303bb12520",
+        "w": 6,
+        "h": 2,
+        "widget": {
+          "size": 0,
+          "dataType": "incidents",
+          "params": {
+            "groupBy": [
+              "owner"
+            ]
+          },
+          "query": "",
+          "modified": "2018-01-25T10:00:37.28790579Z",
+          "name": "dailyIncidentReport Analyst",
+          "dateRange": {
+            "fromDate": "0001-01-01T00:00:00Z",
+            "toDate": "0001-01-01T00:00:00Z",
+            "period": {
+              "byTo": "",
+              "byFrom": "days",
+              "toValue": null,
+              "fromValue": null,
+              "field": ""
+            },
+            "fromDateLicense": "0001-01-01T00:00:00Z"
+          },
+          "isPredefined": false,
+          "version": 1,
+          "id": "113",
+          "widgetType": "bar"
+        }
+      }
+    ],
+    "name": "Daily incidents",
+    "isPredefined": false,
+    "version": 0,
+    "fromDate": "0001-01-01T00:00:00Z",
+    "id": ""
+  },
+  "endingDate": "0001-01-01T00:00:00Z",
+  "sensitive": false,
+  "version": 6,
```

Comment:
I understand you copied a created reports json. Important to note that this marks a custom made report. please remove the version from all reports